Packages/com.unity.inputsystem/InputSystem/Devices/Pointer.cs

```diff
@@ -180,6 +180,11 @@ public class Pointer : InputDevice, IInputStateCallbackReceiver
         /// </remarks>
         public ButtonControl press { get; protected set; }
 
+        /// <summary>
+        /// The index of the display that was pointed.
+        /// </summary>
+        public virtual IntegerControl displayIndex { get; protected set; }
```

Comment:
Why does this need to be virtual? What is the use-case for this? I was under the impression this would be a plain set/get?

Reply:
@3iD0loN can you comment here too? I don't have context on why it's virtual.

Reply:
Looking at the code I cannot see why IntegerControl displayIndex in Pointer cannot be made non-virtual as other controls. In addition to this e.g. map the displayIndex IntegerControl to displayIndex field in FinishSetup() for Mouse and map to underlying primaryTouch displayIndex for TouchScreen? (See examples of existing controls in FinishSetup). Did you try this and it didn't work? Since primaryTouch control always exist in TouchScreen I believe this should be feasible?

Reply:
@jamesmcgill do you have any additional thoughts on this one?

Reply:
Yeah, I would agree with @ekcoh . If it's possible to make it non-virtual using `FinishSetup`, then we should try with that first to keep it consistent.

Reply:
Sure thing, I'll remove the virtual.

Reply:
I was just about to remove virtual, but then I realized it's used in Touchscreen.cs:
```
        public override IntegerControl displayIndex
        {
            get
            {
                return primaryTouch.displayIndex;
            }

            protected set
            {
                primaryTouch.displayIndex = value;
            }
        }
```

I'm not sure how we could replicate that behavior in FinishSetup.  I might reach out on slack to better understand.

Reply:
I just pushed a commit the removes the virtual keyword.
In doing so, it uncovered that Pointer.cs was missing a layout entry (because GetChildControl was not used before), so good catch @ekcoh .

Reply:
@EricFulton Feel free to resolve this conversation if you are happy with outcome. We think its good now.